hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBProfile.java

```diff
@@ -105,8 +105,26 @@ public ColumnFamilyOptions getColumnFamilyOptions() {
       columnFamilyOptions.setCompactionStyle(CompactionStyle.LEVEL);
       return columnFamilyOptions;
     }
+  },
+  // NOTE: just for unit test now, don't use it in production
+  TEST {
```

Comment:
I think we should try to avoid introducing test-specific items in production code.  Why not test with existing `DISK` and `SSD` profiles instead?

Reply:
@adoroszlai oh, it is simply because the ColumnFamilyOptions for DISK & SSD happen to be the same, as I tested, so a new profile TEST is introduced.

Reply:
I don't think they are intended to be the same:

https://github.com/apache/ozone/blob/e658f40a5cf1263a7b8036f9f24bac74c8e90372/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBProfile.java#L104-L106

If they are, then it's probably a bug.

Reply:
Yes, I've read this line that setting the compactionStyle to CompactionStyle.LEVEL, but actually it is just the default style, so the results are still the same.

Reply:
Good point.  How about checking non-equality of `getDBOptions().compactionReadaheadSize()` between the two profiles?  Those _are_ different.

Reply:
@adoroszlai well, I admit that there merely any difference to check the DBOptions or ColumnFamilyOptions, here I intended to check the ColumnFamilyOptions just because it is convenient to use the existing OPTIONS_CACHE in AbstractDatanodeStore.
So to check the DBOptions, I'll try to export the DBOptions as well(or the DBProfile directly), just for test, what do you think?

Reply:
@guihecheng Thanks for checking.  I think providing an accessor for `AbstractDatanodeStore`'s `DBProfile` for the test rather is much less intrusive than the fake profile.